geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/commands/executor/set/AbstractSInterIntegrationTest.java

```diff
@@ -60,9 +64,9 @@ public void sinterstoreErrors_givenTooFewArguments() {
   }
 
   @Test
-  public void testSInter() {
-    String[] firstSet = new String[] {"pear", "apple", "plum", "orange", "peach"};
-    String[] secondSet = new String[] {"apple", "microsoft", "linux", "peach"};
+  public void testSInter_givenIntersection_returnsIntersectedMembers() {
+    String[] firstSet = new String[] {"peach"};
+    String[] secondSet = new String[] {"linux", "peach"};
     String[] thirdSet = new String[] {"luigi", "bowser", "peach", "mario"};
     jedis.sadd("{user1}set1", firstSet);
```

Comment:
The strings `"{user1}set1"` and `"{user1}set2" are used a lot in this class. Could they be extracted to constants please?